### PR TITLE
Fix/csv import export UI

### PR DESCRIPTION
**Pull request by fbuys**
**Description:**

_Closes #35 _

Add icon and hover effect to indicate that the CSV import/export header is clickable.

https://user-images.githubusercontent.com/3785596/119365060-d8f27880-bcaf-11eb-8f1e-d03b49b2527d.mov

**Note:**
Added to gitignore
Had to update mimemagic before I could bundle install.
The additional commits come from: https://github.com/fastruby/points/pull/42

**QA**
Test the CSV import/expansion panel.
Is it obviously clickable?
Does it work when you reload the page?

I will abide by the [code of conduct](https://github.com/fastruby/points/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by bronzdoc — APPROVED** [review]

**Comment by bronzdoc** [comment]
@buys-fran update the `.lock.next` for mimemagic aswell

File: Gemfile.next.lock [hunk 1]
```diff
@@ -162,7 +162,9 @@ GEM
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (1.0.0)
-    mimemagic (0.3.5)
+    mimemagic (0.3.10)
```
Comment (buys-fran) [inline Gemfile.next.lock:165]: Update mimemagic

File: Gemfile.lock [hunk 2]
```diff
@@ -161,9 +161,11 @@ GEM
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (1.0.0)
-    mimemagic (0.3.5)
+    mimemagic (0.3.10)
```
Comment (buys-fran) [inline Gemfile.lock:164]: Update mimemagic

File: app/assets/javascripts/project.js [hunk 3]
```diff
@@ -1,6 +1,12 @@
+document.addEventListener("turbolinks:load", function() {
+  $(".import-export-header").click(function () {
+    $(this).children(".rotate").toggleClass("reverse");
```
Comment (buys-fran) [inline app/assets/javascripts/project.js]: Rotates the icon, might be better to move to more general place?

**Comment by buys-fran** [comment]
@bronzdoc could you check a last time please?
Also can I merge after one approval?

**Comment by buys-fran** [comment]
@torresga do you mind doing the QA for this PR?

**Comment by torresga** [comment]
@buys-fran Nope! I will take a look at this on Monday

**Comment by etagwerker** [comment]
@torresga I just deployed this to staging: https://staging.points.ombulabs.com 

@buys-fran It looks a little weird when you are using Safari. Here is a screen recording: 

https://user-images.githubusercontent.com/17584/121109591-e3e1f880-c7d9-11eb-9daf-c6fdadb1223c.mov

**Comment by buys-fran** [comment]
The additional commits come from: https://github.com/fastruby/points/pull/42

**Comment by lubc** [comment]
@fbuys hey, could you please fix the git conflict so can do a final QA and then merge it? Thanks!

**Comment by fbuys** [comment]
> @fbuys hey, could you please fix the git conflict so can do a final QA and then merge it? Thanks!

Sure @lubc , thanks for checking this PR out. I fixed the conflicts and pushed again :-)

**Review by torresga — APPROVED** [review]
I QAed this - it works/looks as expected to me.

**Pull request merged**